chemprop/v2/cli/train.py

```diff
@@ -341,33 +355,29 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
         type=list[float],
         help="Weights associated with each target, affecting the relative weight of targets in the loss function. Must match the number of target columns.",
     )
+    train_args.add_argument("-l", "--loss-function", action=RegistryAction(LossFunctionRegistry))
     train_args.add_argument(
-        "-l",
-        "--loss-function",
-        action=RegistryAction(LossFunctionRegistry),
-    )
-    train_args.add_argument(
-        "--v-kl", 
+        "--v-kl",
         "--evidential-regularization",
-        type=float, 
+        type=float,
         default=0.2,  # TODO: the default in v1 was 0. Do we want it to default to 0.2 in v2?
```

Comment:
probably best to keep it the same to eliminate any surprises. In principle, however, it would be better to stick with the paper default

Reply:
see #352 for explanation of why 0.0 was chosen as the default in v1 instead of 0.2 from the paper. I think we should stick with 0.0.